rclpy/rclpy/__init__.py

```diff
@@ -56,6 +56,20 @@ def create_node(node_name, *, namespace=None):
 
 
 def spin_once(node, *, timeout_sec=None):
+    """
+    Execute one item of work or wait until timeout expires.
+
+    One callback will be executed in a SingleThreadedExecutor as long as that
+    callback is ready before the timeout expires.
+
+    It is possible the work done may be for a node other than the one passed to this method
+    if the global executor has a partially completed coroutine.
+
+    :param node: A node to add to the executor to check for work.
+    :param timeout_sec: Maximum time in seconds to wait for work.
```

Comment:
Is this blocking forever if negative timeout provided? Is the default behavior to wait forever as well? (I guess it's yes in both questions, but it may be worth mentioning it in the docblock

Reply:
Yup. I copy/pasted the timeout_sec documentation from `executor.spin_once()` in 93e42b571467d241f20f7832bb46ef468640f5de